docroot/modules/custom/cgov_glossary/cgov_glossary.install

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
What is this file for? The install is just loading 2 roles and doing nothing with them. (I actually don't know why this module would need an install hook or even a .install file.

Reply:
I've deleted this file! It was likely copied over from cgov_cts.